### DISCUSSION TITLE
Able to disable actions (mantling, rolling, jumping, sprinting, etc) from BP?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#282] [Q&A]
Just curious if there's a way to disable specific actions like rolling, mantling, sprinting in BP? I'm using the Event On Locomotion Action Changed to do some stuff, but it would be nice to have a way to disable specific actions from working (like jumping; I'd think most games would always have some point where you can't jump?)

**Comment by xkamronx** [comment]
Still curious about this before diving in to add something.

**Comment by oivio** [comment]
why just not remove default inputs from ALS settings for rolling, sprinting? For mantle I do agree, it would be nice to have it separated. Something like ALS-Community do. Having this as sperate component would be nice.
